tasks/orka-init.yaml

```diff
@@ -23,6 +42,34 @@ spec:
       type: string
       description: Enables or disables VNC for the VM.
       default: "true"
+    - name: vm-metadata
+      type: string
+      description: "Inject custom metadata to the VM. You need to provide the metadata in format: {\"items\": [{\"key\": \"\", \"value\": \"\"}]}"
+      default: ""
+    - name: system-serial
+      type: string
+      description: "Assign an owned macOS system serial number to the VM. You need to provide the systemSerial in format: {{systemSerial}}"
+      default: ""
+    - name: io-boost
+      type: string
+      description: Enables or disables IO performance improvements for the VM.
+      default: "false"
+    - name: gpu-passthrough
```

Comment:
for reasoning mentioned elsewhere, can we move this param to the `orka-deploy` task?